app/users/index/controller.js

```diff
@@ -1,8 +1,25 @@
 import AbstractPagedController from 'hospitalrun/controllers/abstract-paged-controller';
+import Ember from 'ember';
 import UserSession from 'hospitalrun/mixins/user-session';
-export default AbstractPagedController.extend(UserSession, {
+import UserRoles from 'hospitalrun/mixins/user-roles';
+
+export default AbstractPagedController.extend(UserSession, UserRoles, {
   addPermission: 'add_user',
   deletePermission: 'delete_user',
-  sortProperties: ['displayName']
-
+  sortProperties: ['displayName'],
+  namedRolesChanged: Ember.observer('namedRoles', function() {
```

Comment:
[Observers are an anti-pattern](https://dockyard.com/blog/2015/11/16/best-practices-functional-programming-and-the-observer-effect) and should be avoided.  We are trying to remove as many observers as possible from HospitalRun and I would like to see this refactored/changed to not use an observer.